src/univariate/continuous/normal.jl

```diff
@@ -74,22 +73,18 @@ kurtosis(d::Normal{T}) where {T<:Real} = zero(T)
 
 entropy(d::Normal) = (log2π + 1)/2 + log(d.σ)
 
-
 #### Evaluation
-
-@_delegate_statsfuns Normal norm μ σ
-
 gradlogpdf(d::Normal, x::Real) = (d.μ - x) / d.σ^2
-
-mgf(d::Normal, t::Real) = exp(t * d.μ + d.σ^2/2 * t^2)
-cf(d::Normal, t::Real) = exp(im * t * d.μ - d.σ^2/2 * t^2)
```

Comment:
I don't see why these should go

Reply:
Thought I had put those back with `gradlogpdf.`

Reply:
They're now below